lib/elixir_console_web/live/console_live/command_input_component.html.leex

```diff
@@ -8,7 +8,6 @@
       autocomplete="off"
       autofocus
       name="command"
-      phx-keydown="keydown"
       phx-target="#command_input"
```

Comment:
Do we still need this `phx-target`?

Reply:
Looks like the app works fine without it, since we're setting the target in the js `pushEventTo`. However, the live tests fail when removing this line because it doesn't use the js at all so the handler is expected to be defined in the live view, not in the component.

Reply:
Maybe this can be solved in LV 0.16 after this commit: https://github.com/phoenixframework/phoenix_live_view/commit/402b7fbb91fa3c85c9d5d07a2afd031bbdda2607

Reply:
Sounds good to me. Just opened https://github.com/wyeworks/elixir_console/pull/141 so we're close.